addons/mail/static/src/js/utils.js

```diff
@@ -194,7 +194,7 @@ function timeFromNow(date) {
     if (moment().diff(date, 'seconds') < 45) {
         return _t("now");
     }
-    return date.fromNow();
+    return moment(date).fromNow();
```

Comment:
this change is not needed if we have a moment as argument which would be the case if we use parseDateTime in the `_updateTimestamps` method